js/stack/stack-dom.js

```diff
@@ -9,29 +9,78 @@ const takeStackBtn = document.getElementById('take-stack');
 const newStack = new StackDataStructure();
 
 const clearStackInput = () => {
-  // ... your code goes here
+
+  stackInput.value = 0
+  
 };
 
 const renderListStack = () => {
-  // ... your code goes here
+
+  for (i = 0; i < newStack.MAX_SIZE; i++) {
+    const li = document.createElement('li')
+    li.setAttribute('class', 'inactive')
+    stackList.appendChild(li)
+  }
 };
 
 renderListStack();
 
 const generateWarningStack = type => {
   if (type === 'underflow') {
-    // ... your code goes here
+    warningBottomStack.innerHTML = 'Underflow'
+    warningBottomStack.style.display  = 'block'
+
   } else if (type === 'overflow') {
-    // ... your code goes here
+    warningTopStack.innerHTML = 'Overflow'
+    warningTopStack.style.display = 'block'
+
   }
 };
 
+
 const addToStack = () => {
-  // ... your code goes here
+  let input = stackInput
+  newStack.push(input)
+  console.log("HOla")
+  
+  let allLi = document.querySelectorAll('li')
+  
+  if (newStack.canPush() === false) {
+    generateWarningStack('overflow')
+
+  } else {
+    for (let i = 0; i <= newStack.stackControl.length - 1; i++) {
+      allLi[i].classList.add('active')
+      allLi[i].classList.remove('inactive')
+      allLi[i].innerHTML = input.value
+    }
+    warningBottomStack.style.display = 'none'
+  }
+
 };
 
-const removeFromStack = () => {
-  // ... your code goes here
+  const removeFromStack = () => {
```

Comment:
En principio el planteamiento es correcto, pero tendrías que volver a evaluar la condición del for, puesto que no está cambiando la clase al elemento de manera ordenada.